rct229/data/ashrae_90_1_table_G3_1_3_7.json

```diff
@@ -0,0 +1,80 @@
+{
+    "heat_rejection_leaving_water_temp":[
+        {
+            "climate_zone": "CZ0A",
+            "degree_Fahrenheit": 80
```

Comment:
Should this field be called "leaving_water_temperature" to match the standard table? Perhaps add the suffix "_dF"?